src/main/java/jpos/config/simple/xml/JavaxRegPopulator.java

```diff
@@ -0,0 +1,450 @@
+        return JavaxRegPopulator.class.getName();
+    }
+
+    @Override
+    public URL getEntriesURL() {
+        URL url = null;
+
+        if (getPopulatorFileURL() != null &&
+                !getPopulatorFileURL().equals(""))
+            try {
+                url = new URL(getPopulatorFileURL());
+            } catch (Exception e) {
+                tracer.println("getEntriesURL: Exception.message=" +
+                        e.getMessage());
+            }
+        else
+            url = createURLFromFile(new File(getPopulatorFileName()));
+
+        tracer.println("getPopulatorFileURL()=" + getPopulatorFileURL());
+        tracer.println("getPopulatorFileName()=" + getPopulatorFileName());
+
+        return url;
+    }
+
+
+    @Override
+    public void save(Enumeration entries)
+            throws Exception {
+        if (isPopulatorFileDefined())
+            save(entries, getPopulatorFileOS());
+        else
+            try (FileOutputStream os = new FileOutputStream(DEFAULT_XML_FILE_NAME)) {
+                save(entries, os);
+            }
+    }
+
+    @Override
+    public void save(Enumeration entries, String fileName)
+            throws Exception {
+        try (FileOutputStream os = new FileOutputStream(fileName)) {
+            save(entries, os);
+        }
+    }
+
+    @Override
+    public void load() {
+        try (InputStream is = isPopulatorFileDefined() ? new FileInputStream(DEFAULT_XML_FILE_NAME) : getPopulatorFileIS()) {
+            load(is);
+        } catch (Exception e) {
+            tracer.println("Error while loading populator file Exception.message: " +
+                    e.getMessage());
+            lastLoadException = e;
+        }
+    }
+
+    @Override
+    public void load(String fileName) {
+        try (InputStream is = new File(fileName).exists() ? new FileInputStream(fileName) : findFileInClasspath(fileName)) {
+            load(is);
+        } catch (Exception e) {
+            tracer.println("Error while loading populator file Exception.message: " +
+                    e.getMessage());
+            lastLoadException = e;
+        }
+    }
+
+    @Override
+    public String getName() {
+        return JAVAX_REG_POPULATOR_NAME_STRING;
+    }
+
+    //--------------------------------------------------------------------------
+    // Private methods
+    //
+
+    private void save(Enumeration<JposEntry> entries, OutputStream outputStream)
+            throws ParserConfigurationException, TransformerFactoryConfigurationError, TransformerException {
+        Document document = createEmptyDocument();
+
+        insertJposEntriesInDoc(entries, document);
+        insertDateSavedComment(document);
+
+        DOMSource source = new DOMSource(document);
+        TransformerFactory factory = TransformerFactory.newInstance();
+  //      factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "file,jar:file");
+  //      factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "file,jar:file");
```

Comment:
I guess, referencing `XMLConstants.ACCESS_EXTERNAL_SCHEMA` is an error in the issue documentation of our security tool. 
It probably should be 

```suggestion
        factory.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "file,jar:file");
```

I run the XML tests with it and it did not fail anymore.